proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -54,6 +55,7 @@ enum ResourceType {
   RESOURCE_TYPE_TOKEN = 10;
   RESOURCE_TYPE_SCIM_TOKEN = 11;
   RESOURCE_TYPE_REPOSITORY_LABEL = 12;
+  RESOURCE_TYPE_SERVER = 13;
```

Comment:
@gilwong00 This PR captures the proto changes for adding an audit log event. 

The goal is to record a "server" change, specifically "breaking changes" updates. The outer payload contains generic things like who did it, timestamp, metadata (tracing, request, etc.) and the payload itself.